src/components/Calculator.js

```diff
@@ -1,14 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Button from './Button';
 import './calculator.css';
+import caclulate from '../logic/calculate';
 
 const btnKeys = ['AC', '+/-', '%', '÷', '7', '8', '9', '×', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
 
```

Comment:
- [ ] I noticed you must have used the wrong multiplication sign which is causing the multiplication operation to fail. When I changed it and used the X sign, it worked. Maybe you copied the sign and pasted it or so. Kindly remove the current 'x' sign and type in X manually from your keyboard and test your app. Please fix that 👍🏽

<img width="1128" alt="Screenshot 2023-07-06 at 7 40 44 AM" src="https://github.com/migo1/math-magicians/assets/50941074/46164db2-3f1b-41e9-8249-9805e976ff63">